package.json

```diff
@@ -22,6 +22,7 @@
   ],
   "dependencies": {
     "body-parser": "~1.15.0",
+    "compression": "^1.6.1",
```

Comment:
I think need to find a minimal acceptable version for this to work. is 1.6.1 latest or minimal version?